### PR TITLE
Feature/appng 2308

### PR DESCRIPTION
**Pull request by stuemke**
Hi,

please review my changes regarding the messages and HTTP status code for datasource called as service.

BR
Claus

### CODE REVIEW

File: appng-xmlapi/src/main/xsd/appng-platform.xsd [hunk 1]
```diff
@@ -771,6 +771,7 @@ Therefore, several {@link Template}s are used, which contain the nessecarry XSL
 			</xs:appinfo>
 		</xs:annotation>
 		<xs:sequence>
+			<xs:element name="messages" type="messages" minOccurs="0" />
```
Comment (madness-inc) [inline appng-xmlapi/src/main/xsd/appng-platform.xsd]: Please place messages after config, same as for <action>

**Comment by madness-inc** [comment]
Hi Claus,

did you also verify that the messages do appear on the right spot in case of a "regular" GUI-call of the datasource?

Regards
MM

File: appng-xmlapi/src/main/xsd/appng-platform.xsd (hunk 1 above)
Comment (stuemke) [reply appng-xmlapi/src/main/xsd/appng-platform.xsd]: I've placed the messages after config

**Comment by stuemke** [comment]
Hi Matthias,

yes, I checked that any messages added to the datasource within a "reqular" GUI-Call are added in page->messages and not within the datasource to guarantee backward compatibility.

Best Regards
Claus

**Pull request merged**